### PR TITLE
updated WrapSphere 

### PR DESCRIPTION
**Pull request by pariterre**
### Brief summary of changes
Updated WrapSphere implementation according to new OpenSim 4.0 API

### Testing I've completed
They should all pass

### Looking for feedback on...
This PR should mimic the previous one on Cylinders

### CHANGELOG.md (choose one)
- no need to update because CHANGELOG.md's text already fits these changes (after PR on Cylinders is accepted)

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/2298)

### CODE REVIEW

File: OpenSim/Simulation/Wrap/WrapSphere.cpp [hunk 1]
```diff
@@ -64,44 +61,28 @@ WrapSphere::~WrapSphere()
...
 //=============================================================================
 //_____________________________________________________________________________
 /**
- * Set the data members of this WrapSphere to their null values.
+ * Connect properties to local pointers.
  */
-void WrapSphere::setNull()
+void WrapSphere::constructProperties()
 {
+    constructProperty_radius(1.0);
```
Comment (chrisdembia) [inline OpenSim/Simulation/Wrap/WrapSphere.cpp]: Consider using a more reasonable default.

**Comment by jimmyDunne** [comment]
@pariterre, can you do update with master and test? If it passes we can merge.

**Comment by pariterre** [comment]
@jimmyDunne Tests seems good

**Comment by jimmyDunne** [comment]
Pulled, built locally, and tested through Matlab. Everything looks good and the xml files have minor comments as differences. 

Thanks, @pariterre!

**Pull request merged**